### ISSUE TITLE
Accessibility Enhancements

### ISSUE THREAD

**Issue by shreyyq** [Akshatchaube01/TimeWarp#174]
**Describe the bug**
The website does not fully adhere to web accessibility standards (WCAG). For instance, images lack alt text, and there are insufficient color contrast ratios in some sections.

**Additional context**
Benefits: Improved usability and inclusivity, potentially increasing the user base.

please assign me this issue under gssoc.
